src/logic/__tests__/validateCurrentProject.test.js

```diff
@@ -0,0 +1,56 @@
+import validateCurrentProject from '../validateCurrentProject';
+
+import Analyzer from '../../analyzers';
+import validateSource from '../helpers/validateSource';
+import {getCurrentProject} from '../../selectors';
+
+const mockHtmlSource = '<html></html>';
+const mockCssSource = 'div {}';
+
+jest.mock('../../analyzers');
+jest.mock('../helpers/validateSource');
+jest.mock('../../selectors', () => ({
+  getCurrentProject: jest.fn(() => ({
+    sources: {
+      html: mockHtmlSource,
+      css: mockCssSource,
+    },
+  })),
+}));
+
+const dummyState = {};
+
+test('should validate current project', async () => {
+  const dispatch = jest.fn();
+  const done = jest.fn();
+  await validateCurrentProject.process(
+    {
+      getState: () => dummyState,
```

Comment:
I think `eslint-plugin-lodash` (introduced after you created this branch) is going to tell you to use `constant` here.